extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeService.java

```diff
@@ -111,7 +115,7 @@ public KioskList getKioskList() throws ExtractionException {
                 public KioskExtractor createNewKiosk(StreamingService streamingService,
                                                      String url,
                                                      String id)
-                        throws ExtractionException {
+                    throws ExtractionException {
```

Comment:
One more tab ;-)
Wraps should be indented with two more steps, otherwise they can be confused with the content below

Reply:
Done.

I really hope we can have a code style file, so I can press autoformat again without worrying.

Reply:
For some reason this in not yet done, even though GitHub told me the was a change on that line. But if on Android Studio you see everything as expected, it is ok. Is not so important ;-)